### ISSUE TITLE
Use newer travis infrastructure

### ISSUE THREAD

**Issue by jhlch** [sparklingpandas/sparklingpandas#132]
In order to use newer travis infra we need to add the line `sudo: false` to our travis config. In order to be able to set sudo to false and still have our tests build, we need to reworkt hem to use Miniconda, as [in this example.](http://conda.pydata.org/docs/travis.html)

**Comment by dgtall** [comment]
Can I start working on this?

**Comment by jhlch** [comment]
Yes! I have a branch that starts doing this, but it was just me poking at it for 5 minutes and it doesn't work yet. Feel free to work off my branch or start your own: https://github.com/sparklingpandas/sparklingpandas/pull/135
